ee/tabby-webserver/src/handler.rs

```diff
@@ -25,6 +25,22 @@ use crate::{
     ui,
 };
 
+pub fn wrap_logger_raw(logger: Arc<dyn EventLogger>) -> Arc<dyn RawEventLogger> {
```

Comment:
why this need to be public? also prefer just inlining it as this is only place it got used..